src/components/ListItem.jsx

```diff
@@ -78,16 +81,52 @@ export function ListItem({ item, listToken }) {
 		}
 	};
 
+	let itemDisplay;
+
+	if (isEditing) {
+		itemDisplay = (
+			<form
+				onSubmit={(e) => {
+					e.preventDefault();
+					setIsEditing(false);
+					try {
+						updateItemName(listToken, item, updatedName);
+					} catch (error) {
+						console.log('error', error);
+					}
+				}}
+			>
+				<input
+					type="text"
+					placeholder="Edit Item"
+					value={updatedName}
+					onChange={(e) => {
+						setUpdatedName(e.target.value);
+					}}
+				/>
+				<button type="submit">Submit</button>
+			</form>
```

Comment:
Oooo! This is so fun! I love the ability to edit. Could we also add a 'cancel' in case a user decides they don't, in fact, want to edit? I've created the code: 

First, make a function: 
	function handleCancelEdit(e) {
		e.preventDefault();
		setIsEditing(false);
		setUpdatedName('');
	}

Then, create a new button (below the submit button): 
<button onClick={handleCancelEdit}>Cancel</button>

Since this is all within the isEditing state, the cancel button will only show after a user clicks 'edit'. Try it! Let me know what you think!

Reply:
That's a great idea to give a chance to abort the edit! I like it!

Reply:
I will work on this later today or tomorrow, thanks for the suggestions!